common4j/src/test/com/microsoft/identity/common/java/dto/AccessTokenTest.java

```diff
@@ -53,9 +54,13 @@ public void testShouldRefreshWhileStillValid() {
     }
 
     @Test
-    public void testShouldRefreshWhenNoPropertiesAreSet() {
+    public void testShouldNotRefreshWhenNoPropertiesAreSet() {
         final AccessTokenRecord accessToken = new AccessTokenRecord();
-        assertTrue(accessToken.shouldRefresh());
+        try{
+            accessToken.shouldRefresh();
+        } catch (Exception e){
+            assertTrue(e instanceof NumberFormatException);
```

Comment:
Are we looking for the exception in this test or not? It is not clear. I mean the test should fail if it doesn't get the exception and it was expecting one

Reply:
There should be a NumberFormatException if the AccessToken isn't initialized.